data/migrations/20211103214447_newsfeed.js

```diff
@@ -1,13 +1,14 @@
-exports.up = (knex) => {
-  return knex.schema.createTable('newsfeed', function (table) {
-    table.increments('newsfeed_id');
-    table.string('title').notNullable();
-    table.string('link').notNullable();
-    table.text('description').notNullable();
-    table.timestamp('posted_at').defaultTo(knex.fn.now());
-  });
+exports.up = (knex) => {/* eslint-disable */
```

Comment:
Let's not disable disable our linter.

Reply:
Please run the `npm lint-fix` command to format your documents properly in lieu of disabling the linter.